lib/existing-eks-opensource-observability-pattern/index.ts

```diff
@@ -81,6 +83,12 @@ export default class ExistingEksOpenSourceobservabilityPattern {
         console.log(doc);
         fs.writeFileSync(__dirname + '/../common/resources/otel-collector-config-new.yml', doc);
 
+        if (utils.valueFromContext(scope, "adotcollectormetrics.pattern.enabled", false)) {
```

Comment:
Please copy the entire doc handling logic from New OSS pattern. This is not consistent with OSS New Pattern.